src/DetailsView/components/issue-filing-dialog.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Dialog, DialogFooter, DialogType } from 'office-ui-fabric-react/lib/Dialog';
+import * as React from 'react';
+
+import { BugFilingService } from '../../bug-filing/types/bug-filing-service';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { BugServiceProperties } from '../../common/types/store-data/user-configuration-store';
+import { ActionAndCancelButtonsComponent } from './action-and-cancel-buttons-component';
+import { NamedSFC } from '../../common/react/named-sfc';
+
+export interface IssueFilingDialogProps {
+    deps: IssueFilingDialogDeps;
+    isOpen: boolean;
+    selectedBugFilingService: BugFilingService;
+    selectedBugData: CreateIssueDetailsTextData;
+    selectedBugFilingServiceData: BugServiceProperties;
+    bugFileTelemetryCallback: (ev: React.SyntheticEvent) => void;
+    onClose: (ev: React.SyntheticEvent) => void;
+}
+
+export interface IssueFilingDialogDeps {}
+
+const titleLabel = 'Specify issue filing location';
+
+export const IssueFilingDialog = NamedSFC<IssueFilingDialogProps>('IssueFilingDialog', props => {
```

Comment:
Two questions:
- This is not being use anywhere at this moment, right?
- I don't see the different issue filing options being rendered here, just the buttons to 'cancel' and 'file issue'. Am I missing something?

Reply:
1. Correct, not yet.
2. Yes, because the content/container for that content is a different control (since that will be re-used in settings as well).